synedrion/src/cggmp21/aux_gen.rs

```diff
@@ -626,6 +626,7 @@ impl<P: SchemeParams, I: PartyId + Serialize> Round<I> for Round3<P, I> {
     }
 }
 
+#[allow(clippy::indexing_slicing)]
```

Comment:
We can probably deal with it the same as with `unwrap_used`, `#![cfg_attr(not(test), warn(clippy::indexing_slicing))]` in `lib.rs`